### ISSUE TITLE
Couldn't find Nim version

### ISSUE THREAD

**Issue by andreaferretti** [nim-lang/nimble#125]
I have installed Nim on Ubuntu 15.04 using the instructions on the Nim Github page (i.e. using Koch, and then adding `$NIMHOME/bin` to the `$PATH`).

I am now trying to install nimble, but when I do `nim c -r src/nimble install`, the process fails with `Couldn't find Nim version.`.

Apparently, the issue stems from the fact that running `osproc.execCmdEx("nim -v")` fails with

    (output: /bin/sh: 1: nim: not found
    , exitCode: 127)

I am not sure how to fix this, as the `nim` command is certainly in the `$PATH` (and in fact `nim -v` works correctly in the same shell).

**Comment by dom96** [comment]
What happens when you execute ``which nim``?

**Comment by dom96** [comment]
Nimble actually uses findExe to find the path to the Nim executable. Try it and see what it returns.

**Comment by andreaferretti** [comment]
When I do `which nim`, I do not find anything. I have tried

    import os, osproc

    echo findExe("nim")

and it prints `/home/papillon/tools/Nim/bin/nim`, as expected. If then I try

    import os, osproc

    echo execCmdEx("nim -v")
    echo execCmdEx("/home/papillon/tools/Nim/bin/nim -v")

the second command prints the expected output, but the first one fails with

    (output: /bin/sh: 1: nim: not found
    , exitCode: 127)

as I remarked above. It seems that the issue is that nimble finds the correct path for the nim executable, but then tries to start nim without the full path
